Dev

czarekk215 opened the pull request.

czarekk215 committed:
Create .gitkeep

czarekk215 committed:
Add files via upload

czarekk215 committed:
Create .gitkeep

czarekk215 committed:
Add files via upload

czarekk215 committed:
Create .gitkeep

czarekk215 committed:
Add files via upload

adamkrawczyk commented on board/e5_dev_board/sample.yaml:
```diff
@@ -0,0 +1,16 @@
+# This file is provided so that the application can be compiled using Twister,
+# the Zephyr testing tool. In this file, multiple combinations can be specified,
+# so that you can easily test all of them locally or in CI.
+sample:
+  description: Example application
+  name: example-application 
+common:
+  build_only: true
+  integration_platforms:
+    - custom_plank
+    - nucleo_f302r8
+tests:
+  app.default: {}
+  app.debug:
+    extra_overlay_confs:
+      - debug.conf
```
Is this file used for something? if so the platforms are missing e5 board if not needed remove

czarekk215 replied:
Removed

adamkrawczyk replied:
if this is resolved then click resolve conversation

adamkrawczyk commented on board/e5_dev_board/src/main.c:
```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
```
Change this to 
```
/*
 * Copyright (c) WiseVision 2023. All rights reserved.
 */

adamkrawczyk commented on board/e5_dev_board/src/main.c:
```diff
@@ -0,0 +1,116 @@
+}
+
+void send_msg(const struct device *dev, uint8_t *data, int16_t rssi, int8_t snr){
+	char data_send[MAX_DATA_LEN] = {0};
+	int ret;
+	LOG_INF("Sending data");
+	int convertedValue = atoi(data);
```
please use the same naming rule for variables most of the zephyr projects follow the camel_case rule, change the `convertedValue` to converted_value`

adamkrawczyk commented on board/e5_dev_board/CMakeLists.txt:
```diff
@@ -0,0 +1,12 @@
+#-------------------------------------------------------------------------------
+# Zephyr Example Application
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
```
change license to 
```
/*
 * Copyright (c) WiseVision 2023. All rights reserved.
 */
 ```

adamkrawczyk commented on board/e5_dev_board/Kconfig:
```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+#
+# This file is the application Kconfig entry point. All application Kconfig
+# options can be defined here or included via other application Kconfig files.
+# You can browse these options using the west targets menuconfig (terminal) or
+# guiconfig (GUI).
```
if the file was modified change this as well

domrobotec replied:
Have you researched the topic of changing license? I know, that Apache-2.0 provides the code AS IS, but I don't have broad knowledge about licensing. I just want to be safe in this regard, there is nothing wrong with using open source code in the product.

adamkrawczyk commented on board/e5_dev_board/prj.conf:
```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+#
+# This file contains selected Kconfig options for the application.
```
same

adamkrawczyk commented on board/e5_mini/CMakeLists.txt:
```diff
@@ -0,0 +1,12 @@
+#-------------------------------------------------------------------------------
+# Zephyr Example Application
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
```
.

adamkrawczyk commented on board/e5_mini/sample.yaml:
```diff
@@ -0,0 +1,16 @@
+# This file is provided so that the application can be compiled using Twister,
+# the Zephyr testing tool. In this file, multiple combinations can be specified,
+# so that you can easily test all of them locally or in CI.
+sample:
+  description: Example application
+  name: example-application 
+common:
+  build_only: true
+  integration_platforms:
+    - custom_plank
+    - nucleo_f302r8
+tests:
+  app.default: {}
+  app.debug:
+    extra_overlay_confs:
+      - debug.conf
```
is this file needed?

adamkrawczyk commented on board/e5_mini/src/main.c:
```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
```
.

adamkrawczyk commented on board/e5_mini/src/main.c:
```diff
@@ -0,0 +1,238 @@
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(lora_send);
+
+int map(int value, int inMin, int inMax, int outMin, int outMax) {
+    return (value - inMin) * (outMax - outMin) / (inMax - inMin) + outMin;
+}
```
change the function name to something that explains more what this function does.

adamkrawczyk commented on board/e5_mini/src/main.c:
```diff
@@ -0,0 +1,238 @@
+int configureTx(const struct device *const lora_dev)
+{
+	struct lora_modem_config config;
+	config.frequency = 865100000;
```
maybe add a comment that this is eu freqz and what should be for the us

adamkrawczyk commented on board/e5_mini/src/main.c:
```diff
@@ -0,0 +1,238 @@
+
+void send_msg(const struct device *dev, int moisture){
+	int ret;
+	printk("Received moisture value: %d\n", moisture);
+	char data[MAX_DATA_LEN];
+	char moisture_str[MAX_DATA_LEN];
+	snprintf(moisture_str, sizeof(moisture_str), "%.2d", moisture);
```
if possible please use one way of logging. Possibly the zephyr ways of logging `LOG_INF....`

adamkrawczyk commented on board/e5_mini/src/main.c:
```diff
@@ -0,0 +1,238 @@
+	snprintf(moisture_str, sizeof(moisture_str), "%.2d", moisture);
+	memset(data, 0, sizeof(data));
+	strncpy(data, moisture_str, sizeof(data) - 1);
+	/*for (int i = 0; i < strlen(moisture_str) && i < MAX_DATA_LEN; ++i) {
+	    data[i] = moisture_str[i];
+	}*/
```
remove comment

adamkrawczyk commented on board/e5_mini/src/main.c:
```diff
@@ -0,0 +1,238 @@
+	int err;
+	for (size_t i = 0U; i < ARRAY_SIZE(adc_channels); i++) {
+		if (!adc_is_ready_dt(&adc_channels[i])) {
+			printk("ADC controller device %s not ready\n", adc_channels[i].dev->name);
```
.

adamkrawczyk commented on board/e5_mini/src/main.c:
```diff
@@ -0,0 +1,238 @@
+
+		err = adc_channel_setup_dt(&adc_channels[i]);
+		if (err < 0) {
+			printk("Could not setup channel #%d (%d)\n", i, err);
```
.

adamkrawczyk commented on board/e5_mini/src/main.c:
```diff
@@ -0,0 +1,238 @@
+
+void set_led(uint8_t *data_recive, const struct device *gpio1, const struct device *gpio2){
+	int ret;
+	int SetLed = atoi(data_recive);
```
camel_cased?

adamkrawczyk commented on docs/asset/.gitkeep:
```diff
@@ -0,0 +1 @@
+
```
remove this file

czarekk215 committed:
Update README.md

czarekk215 committed:
Delete board/e5_dev_board/.gitkeep

czarekk215 committed:
modified:   board/e5_dev_board/CMakeLists.txt
	modified:   board/e5_dev_board/Kconfig
	modified:   board/e5_dev_board/debug.conf
	modified:   board/e5_dev_board/prj.conf
	deleted:    board/e5_dev_board/sample.yaml
	modified:   board/e5_dev_board/src/main.c
	deleted:    board/e5_mini/.gitkeep
	modified:   board/e5_mini/CMakeLists.txt
	modified:   board/e5_mini/Kconfig
	modified:   board/e5_mini/boards/lora_e5_dev_board.overlay
	modified:   board/e5_mini/debug.conf
	modified:   board/e5_mini/prj.conf
	deleted:    board/e5_mini/sample.yaml
	modified:   board/e5_mini/src/main.c

czarekk215 committed:
Update README.md

czarekk215 committed:
Update README.md

czarekk215 committed:
Update README.md

czarekk215 committed:
Update README.md

czarekk215 committed:
Update README.md

czarekk215 committed:
Update README.md

czarekk215 committed:
Delete docs/asset/.gitkeep

czarekk215 committed:
Create README.md

czarekk215 committed:
Add files via upload

czarekk215 committed:
Update README.md

czarekk215 committed:
Update README.md

czarekk215 committed:
Delete docs/st-link

czarekk215 committed:
Add files via upload

czarekk215 committed:
Update README.md

czarekk215 committed:
Update README.md

czarekk215 committed:
Update README.md

czarekk215 committed:
Update README.md

czarekk215 committed:
Update README.md

czarekk215 committed:
Add files via upload

czarekk215 committed:
Add files via upload

czarekk215 committed:
Add files via upload

czarekk215 committed:
Update README.md

czarekk215 committed:
Update README.md

czarekk215 committed:
Update README.md

czarekk215 committed:
Update README.md

czarekk215 committed:
Update README.md

czarekk215 committed:
Add files via upload

czarekk215 committed:
Add files via upload

czarekk215 committed:
Update README.md

czarekk215 committed:
Add files via upload

czarekk215 committed:
Update README.md

czarekk215 committed:
Add files via upload

czarekk215 committed:
Update README.md

czarekk215 committed:
Add files via upload

czarekk215 committed:
Update README.md

czarekk215 committed:
Delete docs/asset/logs_mini_board.png

czarekk215 committed:
Add files via upload

czarekk215 committed:
Update README.md

czarekk215 committed:
Update README.md

styczen commented on README.md:
```diff
@@ -1 +1,415 @@
\ No newline at end of file
+# Soil moisture system using grove capicty moisture sensor, Wio E5 dev board and Wio E5 mini board
+## Introduction
+![System schema](./docs/asset/project_cezary_1.drawio.png)
```
Add addional line after because new line is appended to image
![image](https://github.com/wise-vision/project-cezary/assets/20014205/4b1e735d-a646-4e9f-8e06-cb0edc72d243)

domrobotec replied:
Also, use the same grammatical structure for all of the bullet points. e.g.:
The soil moisture system includes:
- Reading the soil moisture level via ADC on the mini board.
- Transmitting data about the moisture level via LoRa to the dev board.
- Receiving this data by the dev board and sending instructions to the mini board about which color the RGB LED should light up.

Additionally, in the image above, I assume that the left board is the mini board, but it is not 100% clear. You can add some explanation at the very beginning, e.g., "The system consists of the following hardware: ..." 
Or you can just combine all of these information into one list :smiley:

styczen commented on README.md:
```diff
@@ -1 +1,415 @@
+* West: 1.2.0
+## Preparation
+1. It is necessary to have Zephyr already installed to build or flash application.
+   * Go to the official Zephyr RTOS website (https://www.zephyrproject.org/) to download the latest version.
```
I would add more direct link to download page. When I opened this link, there is no "Download" button visible.

styczen commented on README.md:
```diff
@@ -1 +1,415 @@
+![System schema](./docs/asset/project_cezary_1.drawio.png)
+Soil moisture system includes:
+* Reading soil mositure level via ADC on mini board.
+* Sending data about mositure level via Lora to dev board.
```
Nit:
```suggestion
* Sending data about mositure level via LoRa to dev board.
```

styczen commented on README.md:
```diff
@@ -1 +1,415 @@
+Soil moisture system includes:
+* Reading soil mositure level via ADC on mini board.
+* Sending data about mositure level via Lora to dev board.
+* Dev board recive this data and send order to mini board about, what color rgb led, should lights.
```
```suggestion
* Dev board receive this data and send order to mini board about, what color rgb led, should lights.
```

styczen commented on README.md:
````diff
@@ -1 +1,415 @@
+west flash
+```
+Successful flash should looks like this:
+![Successful flash](./docs/asset/correct_flash.png)
````
Maybe it might be good to add some information about connecting board to computer i.e. if some programmer/debugger is needed or just standard USB cable (and what type: C, micro, mini etc). Probably update graphics from the beginning.

domrobotec replied:
I don't like the idea of attaching screenshots of terminal output.
I can see that the first image corresponds to build process. Maybe text description like this can replace the image:
Build is successful when all of the build jobs are finished. To test, upload the build to the board.

Regarding the second image, I think @adamkrawczyk is a much more experienced person to review.

styczen commented on README.md:
````diff
@@ -1 +1,415 @@
+	return ret;
+}
+```
+This function is for configuarting reception. Lora newtork has few things to confing, but we have only to choose the frequnecy. Frequnecy is diffrent for diffrent regions.
````
```suggestion
This function is for receiving configuration. LoRa network has a few things to configure, but we have to choose only the frequnecy. Frequnecy is different for different regions.
```

domrobotec replied:
ditto

styczen commented on README.md:
````diff
@@ -1 +1,415 @@
+	return ret;
+}
+```
+This function is for configuarting transmission. Like in previous function we have to choose to correct frequnecy.
````
```suggestion
This function is for transmission configuration. Like in previous function, we have to choose to correct frequnecy.
```

styczen commented on README.md:
````diff
@@ -1 +1,415 @@
+
+}
+```
+This function send data 0 if the soil moisture is accurate and 1 if our plant needs water. The correct level of moisture is set for under 30 %. It could be change for your purpose.
````
```suggestion
This function send data 0 if the soil moisture is correct and 1 if our plant needs water. The correct level of moisture is set for under 30 %. It could be change for your purpose.
```
I assume that you meant correct in this case i.e. soil moisture is at the correct level.

styczen commented on README.md:
````diff
@@ -1 +1,415 @@
+	return 0;
+}
+```
+In main function we recev and send messegs form board to bord with 1 second interval.
````
```suggestion
In main function we receive and send messegs form board to board with 1 second interval.
```

styczen commented on README.md:
````diff
@@ -1 +1,415 @@
+    return (value - inMin) * (outMax - outMin) / (inMax - inMin) + outMin;
+}
+```
+This function is used to translet masurments from ADC to 0-100 values. If we want to have correct values of moisture, we have to make a calibration of the sensor. 
````
```suggestion
This function is used to translate measurement from ADC to 0-100 values. If we want to have correct values of moisture, we have to calibrate the sensor. 
```

styczen commented on README.md:
````diff
@@ -1 +1,415 @@
+	LOG_INF("Received data: %s", data_recive);
+}
+```
+This function recevs data from dev board.
````
```suggestion
This function receives data from dev board.
```

styczen commented on README.md:
````diff
@@ -1 +1,415 @@
+
+}
+```
+This function configs gpios outputs for led controling.
````
```suggestion
This function configures gpios outputs for led controlling.
```

styczen commented on README.md:
````diff
@@ -1 +1,415 @@
+	}
+}
+```
+This function is used to configs analog to digital converter.
````
```suggestion
This function is used to configure analog to digital converter.
```

styczen commented on README.md:
```diff
@@ -1 +1,415 @@
+2. Use command `sudo picocom --baud 115200 /dev/"your_usb_port"`
+### On Wio E5 dev board kit logs should looks like this:
+![Wio E5 dev board logs](./docs/asset/logs_dev_board.png)
+* Recive data informs us that data has been recived
```
```suggestion
* Receive data informs us that data has been received
```

styczen commented on README.md:
```diff
@@ -1 +1,415 @@
+### On Wio E5 dev board kit logs should looks like this:
+![Wio E5 dev board logs](./docs/asset/logs_dev_board.png)
+* Recive data informs us that data has been recived
+* Recived data shows value of moisture
```
```suggestion
* Received data shows value of moisture
```

styczen commented on README.md:
```diff
@@ -1 +1,415 @@
+![Wio E5 dev board logs](./docs/asset/logs_dev_board.png)
+* Recive data informs us that data has been recived
+* Recived data shows value of moisture
+* Sending data infroms us that data has been sended
```
```suggestion
* Sending data informs us that data has been sended
```

styczen commented on README.md:
```diff
@@ -1 +1,415 @@
+![Wio E5 dev board logs](./docs/asset/logs_mini_board.png)
+* Data send! infroms us that data has been sended
+* Recive data informs us that data has been recived
+* Recive data shows value sended from dev board: 1 means you need too water the plant and 0 means that your soil moisture is correct
```
```suggestion
* Receive data informs us that data has been received
* Receive data shows value sended from dev board: 1 means you need too water the plant and 0 means that your soil moisture is correct
```

styczen reviewed (changes requested):
README in general is clear. I found few typos.

I just skimmed the code and one thing I would recommend is to use some formatter.  For C/C++ there is [clang-format](https://clang.llvm.org/docs/ClangFormat.html).

If you do not want to format code by hand, you can use [Git hooks](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks) e.g. `pre-commit` or use your editor of choice to format code when you save.

Other code style thing is to choose one style and stick to it. I saw in a few places `camelCase` and in the other `lower_case_with_underscores`

domrobotec commented:
Some general remarks.

When you are working on a particular branch just yourself please use more often: `git commit --amend --no-edit`. You can avoid multiple commits with the same message. Right now there are 48 commits from which more than half have "Update README" message :smile: 
It can sometimes require force pushing, but don't be afraid of it when you are working on your branch. Try not to force push when there are more people working on the same branch.

domrobotec commented on README.md:
```diff
@@ -1 +1,415 @@
+Project is created with:
+* Zephyr: 3.5.0
+* West: 1.2.0
+## Preparation
```
Prerequisites

domrobotec commented on README.md:
````diff
@@ -1 +1,415 @@
+```
+* Ninja:
+```
+sudo apt-get update
````
As this is one preparation process, you don't have to add `sudo apt-get update` for each build tool. I would leave the first one only.

domrobotec commented on README.md:
```diff
@@ -1 +1,415 @@
+   * Go to the official Zephyr RTOS website (https://www.zephyrproject.org/) to download the latest version.
+   * Follow the installation instructions on the website to install Zephyr RTOS on your system.
+2. Install Build Tools:
+* Zephyr RTOS uses build tools like CMake and ninja. You have to follow this commmands to install:
```
I don't like the phrase "You have to follow this commands".
I would change the whole part to:

Zephyr RTOS uses build tools like CMake and Ninja:
```
sudo apt-get update
sudo apt-get install cmake
sudo apt-get install ninja-build
```

If you want to you can change the first line to something like:
Zephyr RTOS uses build tools like CMake and Ninja. Use following commands to install both:

domrobotec commented on README.md:
```diff
@@ -1 +1,415 @@
+## Preparation
+1. It is necessary to have Zephyr already installed to build or flash application.
+   * Go to the official Zephyr RTOS website (https://www.zephyrproject.org/) to download the latest version.
+   * Follow the installation instructions on the website to install Zephyr RTOS on your system.
```
If we redirect somebody to another website, I would also give a link, so that you don't have to look for the right instructions.

domrobotec commented on README.md:
````diff
@@ -1 +1,415 @@
+sudo apt-get -y install picom
+```                                                         
+## E5 dev board setup             
+To build versions for the e5 dev board, you need an unlocked board(this shows how to unlock board https://wisevision.tech/docs/LoRa/LoRa-e5-dev-board-unprotect-memory), connected via UART and st-link.
````
What are "versions" here?

Space before opening the brackets, no comma after brackets.

"connected via UART and st-link."
I am not an expert in the field, but is it obvious to what the board should be connected?  PC in this case?

domrobotec commented on README.md:
````diff
@@ -1 +1,415 @@
+```                                                         
+## E5 dev board setup             
+To build versions for the e5 dev board, you need an unlocked board(this shows how to unlock board https://wisevision.tech/docs/LoRa/LoRa-e5-dev-board-unprotect-memory), connected via UART and st-link.
+1. You need to copy Source code from github into zephyrpoject directory:
````
Don't start every instruction with "you need". Simple: "Copy the source code from github into `/zephyrpoject` directory" is enough.

domrobotec commented on README.md:
````diff
@@ -1 +1,415 @@
+cd /zephyrpoject
+git clone https://github.com/wise-vision/project-cezary.git
+```
+2. You need to set the path to the code file:
````
`cd` is used to change directory. 
Maybe something like:
Navigate to the build directory:
or
Go the the build directory:

Change the "build directory" to whatever this directory refers to :smiley:

domrobotec commented on README.md:
````diff
@@ -1 +1,415 @@
+```
+Successful flash should looks like this:
+![Successful flash](./docs/asset/correct_flash.png)
+5. After build and flash to the board, to run the app you have to press the reset button on board.
````
remove "you have to"

domrobotec commented on README.md:
```diff
@@ -1 +1,415 @@
+5. After build and flash to the board, to run the app you have to press the reset button on board.
+![Wio E5 dev board reset button](./docs/asset/reset_dev_board.png)
+## E5 mini board setup
+To build versions for the e5 mini board, you need an unlocked board(this shows how to unlock board https://wisevision.tech/docs/LoRa/LoRa-e5-dev-board-unprotect-memory), connected via UART and st-link.
```
ditto

domrobotec commented on README.md:
```diff
@@ -1 +1,415 @@
+![Wio E5 dev board reset button](./docs/asset/reset_dev_board.png)
+## E5 mini board setup
+To build versions for the e5 mini board, you need an unlocked board(this shows how to unlock board https://wisevision.tech/docs/LoRa/LoRa-e5-dev-board-unprotect-memory), connected via UART and st-link.
+1. You need to copy Source code from github into zephyrpoject directory:
```
ditto for whole section

domrobotec commented on README.md:
````diff
@@ -1 +1,415 @@
+CONFIG_LORA=y
+CONFIG_CBPRINTF_FP_SUPPORT=y
+```
+These configurations help customize the Zephyr OS for specific hardware and functionality requirements.
````
Place explanation before the code.

domrobotec commented on README.md:
```diff
@@ -1 +1,415 @@
+![Successful flash](./docs/asset/correct_flash.png)
+5. After build and flash to the board, to run the app you have to press the reset button on board.
+![Wio E5 mni board reset button](./docs/asset/reset_mini_board.png)
+## Code of E5 dev board
```
For the whole section, please refactor. First shortly explain what the piece of code does, next place the piece of code, and next add some more details about the code if needed.

domrobotec commented on README.md:
````diff
@@ -1 +1,415 @@
+}
+```
+This function sets color of leds depending on the values send from dev board.
+## Finishing
````
Add newline before each section

domrobotec commented on README.md:
````diff
@@ -1 +1,415 @@
+```
+This function sets color of leds depending on the values send from dev board.
+## Finishing
+If you flashed your board with built programms you can check their logs.
````
programs

domrobotec commented on README.md:
````diff
@@ -1 +1,415 @@
+}
+```
+This function sets color of leds depending on the values send from dev board.
+## Finishing
````
I don't like "Finishing" as the name of the section but I don't have any idea how to name it. @styczen @adamkrawczyk

adamkrawczyk replied:
Maybe `Conclusion` ?

domrobotec commented on README.md:
```diff
@@ -1 +1,415 @@
+* Data send shows value that has been send: 1 means you need too water the plant and 0 means that your soil moisture is correct
+* Blinking led on board informs us that there is connection via uart.
+### On Wio E5 mini board logs should looks like this:
+![Wio E5 dev board logs](./docs/asset/logs_mini_board.png)
```
This is why I don't like pictures of text in the README. The images are not proportional and the size of letters varies a lot. Use:
```sh
blabla 
goo
foo
```

adamkrawczyk replied:
Add extra line here

domrobotec commented on README.md:
```diff
@@ -1 +1,415 @@
+* Blinking led on board informs us that there is connection via uart.
+### On Wio E5 mini board logs should looks like this:
+![Wio E5 dev board logs](./docs/asset/logs_mini_board.png)
+* Data send! infroms us that data has been sended
```
Try to be more careful with grammar :) Also, when you can, try to avoid making the instruction personal. In most cases you don't need to use words like: "us", "you", "your".
"Data Sent!" informs that the data has been sent.
"Receive Data" informs that the data has been received.
"Receive Data" shows the value sent from the dev board: 1 indicates a need for watering the plant, and 0 indicates that the soil moisture is correct.

domrobotec replied:
Change the whole section accordingly

domrobotec commented:
In general the instructions were clear for me with some minor exceptions :)

adamkrawczyk commented:
> Some general remarks.
> 
> When you are working on a particular branch just yourself please use more often: `git commit --amend --no-edit`. You can avoid multiple commits with the same message. Right now there are 48 commits from which more than half have "Update README" message 😄 It can sometimes require force pushing, but don't be afraid of it when you are working on your branch. Try not to force push when there are more people working on the same branch.

i guess it was due to edit on github

adamkrawczyk commented on board/e5_mini/src/main.c:
```diff
@@ -0,0 +1,245 @@
+	return ret;
+}
+
+void send_msg(const struct device *dev, int moisture){
```
keep all fuc names in the same convention use camelCased

domrobotec replied:
https://stackoverflow.com/a/1722518

domrobotec replied:
https://www.theserverside.com/answer/Camel-case-vs-snake-case-Whats-the-difference

domrobotec replied:
I am confused. I assume, whatever we use is fine as long as we are consistent with one convention

adamkrawczyk commented on board/e5_mini/src/main.c:
```diff
@@ -0,0 +1,245 @@
+	LOG_INF("Data sent!");
+
+}
+void recev_msg(const struct device *dev, uint8_t *data_recive, int16_t rssi, int8_t snr){
```
camelCased

adamkrawczyk commented on board/e5_mini/src/main.c:
```diff
@@ -0,0 +1,245 @@
+	LOG_INF("Received data: %s", data_recive);
+}
+
+void gpios_config(const struct device *gpio1, const struct device *gpio2){
```
.

adamkrawczyk commented on board/e5_mini/src/main.c:
```diff
@@ -0,0 +1,245 @@
+}
+const int DryValue = 1950;
+const int WetValue = 1220;
+int adc_val(){
```
camelCased

adamkrawczyk commented on board/e5_mini/src/main.c:
```diff
@@ -0,0 +1,245 @@
+	return moisture;
+}
+
+void set_led(uint8_t *data_recive, const struct device *gpio1, const struct device *gpio2){
```
.

adamkrawczyk commented on board/e5_mini/src/main.c:
```diff
@@ -0,0 +1,245 @@
+
+}
+
+void adc_config(){
```
.

adamkrawczyk commented on board/e5_dev_board/src/main.c:
```diff
@@ -0,0 +1,115 @@
+	LOG_INF("Received data: %s (RSSI:%ddBm, SNR:%ddBm)", data, rssi, snr);
+}
+
+void send_msg(const struct device *dev, uint8_t *data, int16_t rssi, int8_t snr){
```
camelCased

adamkrawczyk commented on board/e5_dev_board/src/main.c:
```diff
@@ -0,0 +1,115 @@
+	return ret;
+}
+
+void recev_msg(const struct device *dev, uint8_t *data, int16_t rssi, int8_t snr){
```
camelCased

czarekk215 committed:
Add files via upload

czarekk215 committed:
modified:   README.md
	modified:   board/e5_dev_board/prj.conf
	modified:   board/e5_dev_board/src/main.c
	modified:   board/e5_mini/boards/lora_e5_dev_board.overlay
	modified:   board/e5_mini/prj.conf
	modified:   board/e5_mini/src/main.c

czarekk215 committed:
modified:   README.md
	modified:   board/e5_dev_board/src/main.c
	modified:   board/e5_mini/src/main.c

czarekk215 committed:
modified:   .gitignore
	modified:   README.md
	modified:   board/e5_dev_board/src/main.c
	modified:   board/e5_mini/src/main.c

czarekk215 committed:
Delete board/e5_dev_board/build directory

adamkrawczyk reviewed (approved):
lgtm
